game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -333,11 +333,21 @@ public void updateTiledMapRenderer()
         this.currentNPCS = game.getNPCS(game.player.getRoom());
     }
 
+    public boolean getChangeMap()
+    {
+        return(changeMap);
```

Comment:
You are right yes. Screens themselves shouldn't really contain any logic so shouldn't need to be tested, however in this case they do contain some logic that should be in a separate class really, because of this I don't think we should bother testing this.